web/src/test/java/org/springframework/security/web/context/HttpSessionSecurityContextRepositoryTests.java

```diff
@@ -628,6 +628,21 @@ public void saveContextWhenTransientAuthenticationWithCustomAnnotationThenSkippe
 		assertThat(session).isNull();
 	}
 
+	@Test
```

Comment:
Will you please add a comment to the top of this test denoting the ticket number like so:

```
// gh-8947
```

Reply:
Alright, that's done